backend/src/api/users/dto/create-user.dto.ts

```diff
@@ -1,9 +1,23 @@
-import { ApiProperty } from '@nestjs/swagger';
 import { IsDefined, IsNotEmpty, IsOptional, IsString } from 'class-validator';
 
 export class CreateUserDTO {
   @IsString()
   @IsNotEmpty()
   @IsDefined()
   readonly username: string;
+
```

Comment:
Fix login crash :warning:

Reply:
Yeah my bad, forgot to add your properties (and it erased the one you defined ^^')